docs/ingestion/native-batch.md

```diff
@@ -1133,8 +1133,14 @@ the [S3 input source](#s3-input-source) or the [Google Cloud Storage input sourc
 
 ### HTTP Input Source
 
-The HTTP input source is to support reading files directly
-from remote sites via HTTP.
+The HTTP input source is to support reading files directly from remote sites via HTTP.
+
+> **NOTE:** The Druid Console runs with the same privileges as the operating system account that runs the Druid processes, for example the Indexer, Middle Manager, and Peon. This means the `HTTPInputSource` has access to data at any location where the Druid process has permissions. For example, using `HTTPInputSource`, a console user has access to internal network locations where the they would be denied access otherwise.
```

Comment:
When I read Console - I think of the Druid web console. This wording is a little incorrect.

The issue is with users who can submit ingestion tasks - not if they have access to the web console. If they can submit ingestion tasks, the ingestion tasks will run as the user running the Druid process, with access to everything pointed out on this line.

Reply:
Thanks @suneet-s I was only thinking of using the web console as the mechanism to submit an ingestion task, so that obviously misses a large category. I'll think on it and make another pass.